src/main/resources/com/mathworks/ci/BuildArtifactAction/summary.jelly

```diff
@@ -12,12 +12,15 @@
         <font color="#EF2929"><h5>Unable to generate a build artifact. </h5></font>
       </j:if>
     </span>
-    <p><b>Tasks run: <font color="Blue">${it.totalCount}</font></b></p>
-    <br>
-    <b>Failed: <font color="red">${it.failCount}</font></b>
-    </br>
-    <br>
-    <b>Skipped: <font color="orange">${it.skipCount}</font></b>
-    </br>
+    <p>
```

Comment:
@mw-hrastega could verify this agian i remember some changes here suggested by you here 
https://github.com/mathworks/jenkins-matlab-plugin/pull/279/commits/2203ba66c9a06b67936d182c6dddf6ed810b181f

Reply:
Checked with Houman offline and he agrees with the change